geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/util/AccumuloSchemaBuilder.scala

```diff
@@ -110,8 +112,8 @@ object AccumuloSchemaBuilder {
       * @param sharing table sharing
       * @return user data builder for call chaining
       */
-    def tableSharing(sharing: Boolean): AccumuloUserDataBuilder =
-      userData(SimpleFeatureTypes.Configs.TABLE_SHARING_KEY, sharing.toString)
+    @deprecated("table sharing is no longer supported")
+    def tableSharing(sharing: Boolean): AccumuloUserDataBuilder = this
```

Comment:
Seems like this should log an error / warning, etc.

Reply:
added